File: app/src/main/java/com/mgaetan89/showsrage/fragment/SettingsAboutLicensesFragment.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

public class SettingsAboutLicensesFragment extends SettingsAboutFragment {
	@Override
	protected int getTitleResourceId() {
		return R.string.licenses;
	}

	@Override
	protected int getXmlResourceFile() {
		return R.xml.settings_about_licenses;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/fragment/SettingsAboutLicensesFragment.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R

class SettingsAboutLicensesFragment : SettingsAboutFragment() {
    override fun getTitleResourceId() = R.string.licenses

    override fun getXmlResourceFile() = R.xml.settings_about_licenses
}
